islandora_batch.drush.inc

```diff
@@ -131,6 +131,32 @@ function drush_islandora_batch_scan_preprocess_validate() {
   if ($encoding_check !== NULL) {
     drush_set_error('INVALID ENCODING', $encoding_check);
   }
+  $type = drush_get_option('type');
+  if ($type != 'zip' xor 'directory') {
```

Comment:
Hmm, this xor logic fails when i use --type=zip.
`--type must be one of "directory" or "zip".zip                                                          [error]`
I'm on drush 6.3.

Also, the other errors barfed a nice all-caps explanation like "INVALID CONTENT MODEL" and this one didn't, I think because it put a non-NULL as the second argument.